macos/library/FLEFileChooserPlugin.m

```diff
@@ -30,11 +30,21 @@
 static NSString *const kPlatformMethodNameKey = @"method";
 static NSString *const kPlatformMethodArgsKey = @"args";
 
-// Method argument keys.
+// Appearance configuration keys for NSSavePanel.
+static NSString *const kPanelTitleKey = @"title";
+static NSString *const kPanelMessageKey = @"message";
+static NSString *const kOKButtonTextKey = @"prompt";
+
+// Method argument keys for NSSavePanel.
+static NSString *const kInitialDirectoryKey = @"initialDirectory";
 static NSString *const kAllowedFileTypesKey = @"allowedFileTypes";
+static NSString *const kFileNameStringValueKey = @"nameFieldStringValue";
```

Comment:
Similarly, something like initialFileName would be clearer here.

Reply:
Done.